starlette/responses.py

```diff
@@ -37,11 +37,12 @@ def __init__(
         self.body = self.render(content)
         self.init_headers(headers)
 
-    def render(self, content: typing.Union[str, bytes, None]) -> bytes:
+    def render(self, content: typing.Any) -> bytes:
         if content is None:
             return b""
         if isinstance(content, bytes):
             return content
+        assert isinstance(content, str)
```

Comment:
Can we just add the `type: ignore` on line 46 for now? If someone complains, we change it.